includes/wf_crm_webform_postprocess.inc

```diff
@@ -1260,7 +1260,8 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       if (is_array($data) && !empty($data['case'][1]['client_id'])) {
         $params = $data['case'][1];
         // Set some defaults in create mode
-        if (empty($this->ent['case'][$n]['id'])) {
+        // Create duplicate case based on existing case if 'duplicate_case' checkbox is checked
+        if (empty($this->ent['case'][$n]['id']) || $this->data['case'][$n]['duplicate_case'] == 1) {
```

Comment:
For safety let's just change this line to

    if (empty($this->ent['case'][$n]['id']) || !empty($this->data['case'][$n]['duplicate_case'])) {

And then I think we're good to merge :)